WBooks/Models/RentResponse.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RentResponse.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 28/03/2022.
+//
+
+import Foundation
+
+public struct RentResponse: Codable {
+     let from: String
```

Comment:
Declare by group the same data types.
Example: 
let id
     bookID,
     userID: Int